docs/Team/Working-Remote.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: default
+title: "Tallwave Guidelines — Team — Working Remote"
+---
+
+[&larr; Home]({{ site.root }}/) &mdash; [Team]({{ site.root }}/Team)
+
+# Our remote environment
+
+Tallwave as an organization is primarily on-site, but we have a handful of remote employees, as well as a strong culture of working from home when you need to go heads-down on a problem. Currently, ~20% of the full-time dev team and the majority of our contractors are remote. Therefore, async communication is very important, for which we rely on Basecamp. Email is used mainly for company announcements and project-client communication, and video calls are for anything from client meetings to 1-on-1s to pair programming.
+
+As an organization, we are working to make remote more integrated.
+
+Things we currently do are:
+* ensure company meetings have good audio for online attendees
```

Comment:
Capitalize bullet sentences and end with periods. 😃

Reply:
@swilliams does that include the "header" bullets, like `* Set your notification preferences`?

Reply:
If they're headers they should be header tags (and then title-cased). Like `## Set Your Notification Preferences`.